htdocs/web_portal/GOCDB_monitor/index.php

```diff
@@ -1,12 +1,14 @@
-<?php
-
-echo("<br/><H1 align='center'>GOCDB MONITOR</H1><hr/>");
+<br>
+<h1 align='center'>GOCDB MONITOR</h1>
+<hr>
 
+<?php
 require_once __DIR__ . '/tests.php';
-echo "URL's as defined by local_info.xml<br>";
-echo "PI URL is: ".PI_URL ."<br>";
-echo "Portal URl is: ".PORTAL_URL."<br>";
-echo "Server Base URL is: ".SERVER_BASE_URL."<br>";
+
+echo "<p>URLs as defined by local_info.xml</p>";
```

Comment:
So `echo`ing is just required whenever you're using PHP?

Reply:
Yup, print works too but is marignally slower.